### DISCUSSION TITLE
Directory of TipTap apps

### DISCUSSION THREAD

**Discussion by justinmoon** [ueberdosis/tiptap#2363] [Feature Requests]
Besides reading your *excellent* documentation, I found it very instructive to read [Notebag](https://github.com/pretzelhands/notebag) and [GitLab](https://gitlab.com/gitlab-org/gitlab/-/tree/master/app/assets/javascripts/content_editor) source code. 

It would be nice to have a list of these repos somewhere. Perhaps this already exists but I haven't seen it.

**Comment by hanspagel** [comment]
I think I have some bookmarks flying around. We should probably put together a "Awesome Tiptap" list (or update the existing one).

Would you be up to send a PR for the links you mentioned? I’ll put mine in then too.

https://github.com/ueberdosis/awesome-tiptap

**Comment by justinmoon** [comment]
Done https://github.com/ueberdosis/awesome-tiptap/pull/1

**Comment by bdbch** [comment]
I've thought about building a simple site for community content like extensions, showcases, etc. under extensions.tiptap.dev or community.tiptap.dev

Nothing super specific right now - did small tests since I think it would be very beneficial for the community to be able to quickly find extensions they need or check if something already exists without browsing through awesome lists, npm or Github on their own.

I'll update you if we have something more specific.
